src/contracts/core/AVSDirectory.sol

```diff
@@ -48,21 +56,200 @@ contract AVSDirectory is
+     * @param standbyParams The new standby parameters for the operator.
+     * @param operatorSignature If non-empty, the signature of the operator authorizing the modification.
+     *                  If empty, the `msg.sender` must be the operator.
+     */
+    function updateStandbyParams(
+        address operator,
+        StandbyParam[] calldata standbyParams,
+        SignatureWithSaltAndExpiry calldata operatorSignature
+    ) external {
+        if (operatorSignature.signature.length == 0) {
+            // Assert caller is `operator` if signature length is zero.
+            require(msg.sender == operator, "AVSDirectory.updateStandbyParams: invalid signature");
+        } else {
+            // Assert operator's signature has not expired.
+            require(
+                operatorSignature.expiry >= block.timestamp,
+                "AVSDirectory.updateStandbyParams: operator signature expired"
+            );
+            // Assert operator's signature cannot be replayed.
+            require(
+                !operatorSaltIsSpent[operator][operatorSignature.salt], "AVSDirectory.updateStandbyParams: salt spent"
+            );
+            // Assert operator's signature is valid.
+            EIP1271SignatureUtils.checkSignature_EIP1271(
+                operator,
+                calculateUpdateStandbyDigestHash(standbyParams, operatorSignature.salt, operatorSignature.expiry),
+                operatorSignature.signature
+            );
+            // Spend salt.
+            operatorSaltIsSpent[operator][operatorSignature.salt] = true;
+        }
+
+        for (uint256 i; i < standbyParams.length; ++i) {
+            onStandby[standbyParams[i].operatorSet.avs][operator][standbyParams[i].operatorSet.id] =
+                standbyParams[i].onStandby;
 
+            emit StandbyParamUpdated(operator, standbyParams[i].operatorSet, standbyParams[i].onStandby);
+        }
+    }
 
     /**
-     * @notice Called by the AVS's service manager contract to register an operator with the avs.
-     * @param operator The address of the operator to register.
-     * @param operatorSignature The signature, salt, and expiry of the operator's signature.
+     *  @notice Called by AVSs to add an operator to an operator set.
+     *
+     *  @param operator The address of the operator to be added to the operator set.
+     *  @param operatorSetIds The IDs of the operator sets.
+     *  @param operatorSignature The signature of the operator on their intent to register.
+     *
+     *  @dev msg.sender is used as the AVS.
+     *  @dev The operator must not have a pending deregistration from the operator set.
+     *  @dev If this is the first operator set in the AVS that the operator is
+     *  registering for, a OperatorAVSRegistrationStatusUpdated event is emitted with
+     *  a REGISTERED status.
      */
-    function registerOperatorToAVS(
+    function registerOperatorToOperatorSets(
```

Comment:
Wordy. I know this fits with `registerOperatorToAVS` below, but I'm not a fan of the word "Operator" twice in the function name - makes parsing harder.

Suggestions: 
* `registerToOperatorSets`
* `registerOperatorToSets`
* `addOperatorToSets`
* `addToOperatorSets`

Reply:
Will think on this one a bit more...